aten/src/ATen/native/UpSample.h

```diff
@@ -47,6 +49,8 @@
 
 namespace at::native {
 
+using namespace vec;
```

Comment:
I'm not a big fan of this, just to prevent namespace pollution. Would it be very inconvenient to remove this? Especially since it's in a header file.

Reply:
Modified, please review again, thanks!